stable/datadog/templates/deployment.yaml

```diff
@@ -16,6 +16,9 @@ spec:
       labels:
         app: {{ template "datadog.fullname" . }}
         type: deployment
+{{- if .Values.podLabels }}
+{{ toYaml .Values.podLabels | indent 8 }}
+{{- end }}
```

Comment:
Could you please remove them from the deployment? It's on feature freeze and will be deprecated in a few versions.

You can add it to the [clusterchecks deployment](https://github.com/helm/charts/blob/master/stable/datadog/templates/agent-clusterchecks-deployment.yaml) if you want, maybe by splitting the setting to have `.Values.clusterchecksDeployment.podLabels` and `.Values.daemonset.podLabels`.

Reply:
OK removed from the deployment. I've left the clusterchecks thing as I don't know what it is.